src/adzerk/boot_cljs.clj

```diff
@@ -66,7 +64,7 @@
      :output-path output-path
      :cljs-opts   (merge base-opts
                          (when source-map smap-opts)
-                         (when node-target {:target :nodejs}))}))
+                         (dissoc compiler-opts :output-dir :optimizations))}))
```

Comment:
Maybe it makes sense to use `compiler-opts` as first map passed to `merge` so those keys will be overwritten automatically? Will need some extra handling as well though so maybe there's not a big benefit. 

Oh, and now that I read it again, `:output-to` and `:source-map-path` need to be blacklisted as well.